vitest.config.ts

```diff
@@ -1,34 +1,30 @@
-/// <reference types="vitest" />
-import { defineConfig } from 'vite'
-import react from '@vitejs/plugin-react'
-import path from 'path'
+import { defineConfig, mergeConfig } from 'vite';
 
-// https://vitejs.dev/config/
-export default defineConfig({
-    plugins: [react()],
-    resolve: {
-        alias: {
-            '@': path.resolve(__dirname, 'src')
-        }
-    },
```

Comment:
used `mergeConfig` and the config defined in `vite.config.ts` so we only need to change the `vite` config in one place and have it reflect here too